### DISCUSSION TITLE
Fusioncharts Chart Widget Implementation

### DISCUSSION THREAD

**Discussion by eastonmeth** [Laravel-Backpack/community-forum#894] [Q&A (Help)]
Hi, I'm struggling to convert my Chart.js chart widgets over to Fusioncharts, does anybody know the syntactic differences and/or anything I need to do to get this working. 

This is my chart controller that works when I have `use ConsoleTVs\Charts\Classes\Chartjs\Chart;`, but not when I replace it with `use ConsoleTVs\Charts\Classes\Fusioncharts\Chart;`:
```
<?php

declare(strict_types=1);

namespace App\Http\Controllers\Charts;

use Backpack\CRUD\app\Http\Controllers\ChartController;
use ConsoleTVs\Charts\Classes\Fusioncharts\Chart;

class ExampleChartController extends ChartController
{
    public function setup(): void
    {
        $this->chart = new Chart();
        $this->chart->labels(['Label 1', 'Label 2', 'Label 3']);
        $this->chart->load(backpack_url('charts/example'));
    }

    public function data(): void
    {
        $this->chart->dataset('Group 1', 'bar', [1, 2, 3]);
    }
}
```

**Comment by jcastroa87** [comment]
Hello @eastonmeth 

Sorry for super late answer, but did you have console error or any error in log?

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity, I will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
